packages/dbml-core/__tests__/parser/dbml-parse/input/default_tables.in.dbml

```diff
@@ -1,7 +1,7 @@
 Table orders {
   id int [pk, default: 123]
   user_id int [not null, unique]
-  status varchar [default: "Completed"]
+  status varchar [default: 'Completed']
```

Comment:
The output must be the same so I don't think we need to change the string quote.